src/CoreBundle/Entity/UserRelCourseVote.php

```diff
@@ -25,13 +25,35 @@ class UserRelCourseVote
      */
     protected $cId;
 
+//    /**
+//     * @var int
+//     *
+//     * @ORM\Column(name="user_id", type="integer", nullable=false)
+//     */
+//    protected $userId;
     /**
-     * @var int
+     * @ORM\OneToOne (targetEntity="Chamilo\CoreBundle\Entity\User",
+     *      inversedBy="user_rel_course_vote")
```

Comment:
`user_rel_course_vote` is incorrect for `inversedBy`. Because it should be the class property in User entity.